service/resource/configmap/service.go

```diff
@@ -231,6 +231,12 @@ func (s *Service) GetDeleteState(obj, currentState, desiredState interface{}) (i
 	return deleteState, nil
 }
 
+func (s *Service) GetUpdateState(obj, currentState, desiredState interface{}) (interface{}, interface{}, interface{}, error) {
+	// this is a simple resource not concerned with being updated, just fulfilling
```

Comment:
These comments come from https://github.com/giantswarm/operatorkit/blob/master/framework/spec.go#L57

Reply:
Returning nil values makes sense since this resource isn't updated. Nitpick but these could be godoc comments.

Reply:
Very good point, done